.drone.yml

```diff
@@ -1,35 +1,43 @@
-matrix:
-  NAME:
-    - ukhomeofficedigital/lev-ci
-  MAJOR:
-    - 3
-  MINOR:
-    - 5
-  PATCH:
-    - 1
-  DOCKER_USERNAME:
-    - ukhomeofficedigital+lev_ci
-  DOCKER_REPO:
-    - quay.io
-pipeline:
-  docker-build:
-    image: ukhomeoffice/drone-docker
-    repo: '${DOCKER_REPO}/${NAME}'
+---
+kind: pipeline
+name: matrix-1
+type: kubernetes
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: docker-build
+  pull: if-not-exists
+  image: plugins/docker
+  settings:
     dry_run: true
-    when:
-      event: pull_request
-  docker-build-and-push:
-    image: ukhomeoffice/drone-docker
-    secrets: [ docker_password ]
-    username: '${DOCKER_USERNAME}'
-    repo: '${DOCKER_REPO}/${NAME}'
-    registry: '${DOCKER_REPO}'
+    repo: quay.io/ukhomeofficedigital/lev-ci
+  when:
+    event:
+    - pull_request
+
+- name: docker-build-and-push
+  pull: if-not-exists
+  image: plugins/docker
+  settings:
+    registry: quay.io
+    repo: quay.io/ukhomeofficedigital/lev-ci
     tags:
```

Comment:
I think you need to change the tags, as the variables they used to use (defined in the matrix) no longer exist.